### PR TITLE
Send protobuf tracing to frontend instead of json

### PR DESCRIPTION
**Pull request by daniel-wer**
The frontend now requests the tracing from the `/getProto` endpoint, and parses the protobuffer directly, which speeds up the loading of large tracings. The protoToJson functionality is no longer needed and can be removed by @fm3 as part of this PR (we can also remove the /getProto from the route then again imo).
[Edit by @fm3: done that]

### note that the datastore needs to be upgraded on deploy

I also noticed that we now have 3 types of bounding boxes in the frontend again, because protobuffers don't support arrays :/

I'll do and include some speed measurements later :)

### Mailable description of changes:
 - Optimized the loading of large tracings.

### Steps to test:
- Open new/existing skeleton/volume tracings - everything should work as expected.

### Issues:
- part of #2639

------
- [x] Ready for review

### CODE REVIEW

File: app/assets/javascripts/oxalis/model.js [hunk 1]
```diff
@@ -163,13 +170,16 @@ export class OxalisModel {
     if (annotation != null) {
       // Make flow happy
       const nonNullAnnotation = annotation;
-      tracing = await doWithToken(token =>
-        Request.receiveJSON(
-          `${nonNullAnnotation.dataStore.url}/data/tracings/${nonNullAnnotation.content.typ}/${
+      const annotationType = nonNullAnnotation.content.typ;
```
Comment (philippotto) [inline app/assets/javascripts/oxalis/model.js]: Would it make sense to extract lines 173 to 182 into a function in `admin_rest_api`?

**Comment by normanrz** [comment]
Would it be feasible to convert `Point3` to `Vector3` upon loading?

**Comment by daniel-wer** [comment]
To add some performance measurements for a ~680,000 node tracing

Locally:
- json: not possible (java.outofmemoryerror)
- protobuf: 4.3s waiting + 0.5s downloading + 2.2s parsing/decoding = 7s

Dev-Deployment:
- json: 7s waiting + 3.6s download + 4.6s parsing/decoding = 15.2s
- protobuf: 1s waiting + 2.6s download + 2.2s parsing/decoding = 5.8s

Roughly a 3x speedup for the tracing fetching and parsing :)

**Comment by daniel-wer** [comment]
@normanrz What exactly do you mean by that? Currently `Point3` is converted to `Vector3` when the tracing is persisted in the store initially. Which is basically right after the tracing was fetched and parsed.

**Comment by normanrz** [comment]
@daniel-wer alright. I was confused, sorry

**Pull request merged**